torch/optim/adamax.py

```diff
@@ -147,7 +153,8 @@ def adamax(params: List[Tensor],
            beta1: float,
            beta2: float,
            lr: float,
-           weight_decay: float):
+           weight_decay: float,
+           maximize: bool):
```

Comment:
We want to make this default to False otherwise this would be a BC-breaking change.